### PR TITLE
Update & Fix Japanese.

### PR DESCRIPTION
**Pull request by reindex-ot**

### CODE REVIEW

**Comment by xb2016** [comment]
Thank you for providing the translation. However, the Japanese translation file is located at `app/src/main/res/values-ja/strings.xml`. Please correct it and resubmit.

**Comment by reindex-ot** [comment]
Maybe I was tired, but I chose the wrong directory.
Sorry.

**Review by xb2016 — CHANGES_REQUESTED** [review]
Since I don’t understand Japanese, I used ChatGPT to review the translation.
Please confirm if the reviewed content needs any changes.

Also, I noticed that a half-width colon is used consistently in the translation, is this common in Japanese?

Thank you once again for your contribution to the project.

File: app/src/main/res/values-ja/strings.xml [hunk 1]
```diff
@@ -13,147 +14,176 @@
...
     <string name="password">パスワード</string>
     <string name="sign_in">ログイン</string>
     <string name="register">登録</string>
-    <string name="sign_in_via_webview">Webでログイン</string>
-    <string name="sign_in_via_cookies">Cookieでログイン</string>
-    <string name="tourist_mode">ログインをスキップ</string>
-    <string name="error_username_cannot_empty">ユーザー名を入力してください。</string>
-    <string name="error_password_cannot_empty">パスワードを入力してください。</string>
-    <string name="sign_in_failed">ログインに失敗しました。</string>
+    <string name="sign_in_via_webview">WebView でログイン</string>
```
[the remark this answers is not in the document]
Comment (reindex-ot) [reply app/src/main/res/values-ja/strings.xml:25]: ❌️

File: app/src/main/res/values-ja/strings.xml [hunk 2]
```diff
@@ -13,147 +14,176 @@
...
+    <string name="sign_in_failed">ログインに失敗しました</string>
+    <string name="sign_in_failed_tip">この問題が解決しない場合は「WebView でログイン」か「Cookie でログイン」をお試しください。</string>
     <string name="get_it">了解</string>
     <!-- Cookie Sign In Scene -->
-    <string name="cookie_explain">Cookiesはブラウザに保存した特殊なデータです。よくわからない場合はGoogleで検索してください。\n\nCookiesを入力してログインしてください。</string>
-    <string name="text_is_empty">テキストを入力してください</string>
-    <string name="wrong_cookie_warning">入力したCookieは正しくないようです。確認してください。</string>
-
+    <string name="cookie_explain">Cookie とは、ウェブブラウザに保存される小さなデータのことです。Cookie が何なのか、または取得方法がわからない場合は Google で検索をしてください。\n\nログインをするには特別な Cookie を入力してください。</string>
+    <string name="from_clipboard">クリップボードから</string>
```
[the remark this answers is not in the document]
Comment (reindex-ot) [reply app/src/main/res/values-ja/strings.xml:35]: ❌️

File: app/src/main/res/values-ja/strings.xml [hunk 3]
```diff
@@ -13,147 +14,176 @@
   ~ See the License for the specific language governing permissions and
   ~ limitations under the License.
   -->
-
 <resources>
+    <!-- App Name -->
     <!-- Login Scene -->
-    <string name="app_waring">このアプリの内容はインターネットから取得したものです。その内容の一部は使用者のお体や心理に傷つける恐れがあります。</string>
+    <string name="app_waring">このアプリの内容はインターネットから取得したものです。その内容の一部は心身に悪影響を与える可能性があります。\n使用を続行することで規約に同意したとみなされます。</string>
```
[the remark this answers is not in the document]
Comment (reindex-ot) [reply app/src/main/res/values-ja/strings.xml:20]: ❌️

File: app/src/main/res/values-ja/strings.xml [hunk 4]
```diff
@@ -13,147 +14,176 @@
...
+    <string name="error_password_cannot_empty">パスワードは空欄にできません</string>
+    <string name="sign_in_failed">ログインに失敗しました</string>
+    <string name="sign_in_failed_tip">この問題が解決しない場合は「WebView でログイン」か「Cookie でログイン」をお試しください。</string>
     <string name="get_it">了解</string>
     <!-- Cookie Sign In Scene -->
-    <string name="cookie_explain">Cookiesはブラウザに保存した特殊なデータです。よくわからない場合はGoogleで検索してください。\n\nCookiesを入力してログインしてください。</string>
-    <string name="text_is_empty">テキストを入力してください</string>
-    <string name="wrong_cookie_warning">入力したCookieは正しくないようです。確認してください。</string>
-
+    <string name="cookie_explain">Cookie とは、ウェブブラウザに保存される小さなデータのことです。Cookie が何なのか、または取得方法がわからない場合は Google で検索をしてください。\n\nログインをするには特別な Cookie を入力してください。</string>
```
[the remark this answers is not in the document]
Comment (reindex-ot) [reply app/src/main/res/values-ja/strings.xml:34]: ❌️

File: app/src/main/res/values-ja/strings.xml [hunk 5]
```diff
@@ -13,147 +14,176 @@
...
-    <string name="error_username_cannot_empty">ユーザー名を入力してください。</string>
-    <string name="error_password_cannot_empty">パスワードを入力してください。</string>
-    <string name="sign_in_failed">ログインに失敗しました。</string>
+    <string name="sign_in_via_webview">WebView でログイン</string>
+    <string name="sign_in_via_cookies">Cookie でログイン</string>
+    <string name="tourist_mode">ログインをせずに続行</string>
+    <string name="error_username_cannot_empty">ユーザー名は空欄にできません</string>
+    <string name="error_password_cannot_empty">パスワードは空欄にできません</string>
+    <string name="sign_in_failed">ログインに失敗しました</string>
+    <string name="sign_in_failed_tip">この問題が解決しない場合は「WebView でログイン」か「Cookie でログイン」をお試しください。</string>
```
[the remark this answers is not in the document]
Comment (reindex-ot) [reply app/src/main/res/values-ja/strings.xml:31]: ❌️

File: app/src/main/res/values-ja/strings.xml [hunk 6]
```diff
@@ -262,205 +322,292 @@
...
+    <string name="settings_eh_list_mode">リストモード</string>
+    <string name="settings_eh_list_mode_detail">詳細</string>
     <string name="settings_eh_list_mode_thumb">サムネイル</string>
-    <string name="settings_eh_detail_size">詳細情報のサイズ</string>
-    <string name="settings_eh_thumb_size">サムネイルサイズ</string>
-    <string name="settings_eh_show_jpn_title">日本語タイトルを表示</string>
+    <string name="settings_eh_detail_size">詳細情報の幅</string>
+    <string name="settings_eh_list_tile_thumb_size">詳細モードのサムネイルのサイズ</string>
+    <string name="settings_eh_thumb_size">サムネイルモードのサムネイルのサイズ</string>
+    <string name="settings_eh_thumb_show_title">タイトル表示のサムネイルモード</string>
```
[the remark this answers is not in the document]
Comment (reindex-ot) [reply app/src/main/res/values-ja/strings.xml:409]: ❌️

File: app/src/main/res/values-ja/strings.xml [hunk 7]
```diff
@@ -262,205 +322,292 @@
...
     <string name="settings_eh_gallery_site">ギャラリーサイト</string>
-    <string name="settings_eh_launch_page">アプリ既定画面</string>
-    <string name="settings_eh_list_mode">リスト表示モード</string>
-    <string name="settings_eh_list_mode_detail">詳細情報</string>
+    <string name="settings_eh_image_limits">画像の制限</string>
+    <string name="settings_eh_image_limits_summary">読み込み中…</string>
+    <string name="settings_eh_image_limits_summary_ip">"IP ベースの制限: "</string>
+    <string name="settings_eh_image_limits_summary_ip_ok">制限なし</string>
+    <string name="settings_eh_image_limits_summary_ip_restricted">低解像度の画像</string>
+    <string name="settings_eh_image_limits_summary_acc">現在: %1$d / %2$d</string>
```
[the remark this answers is not in the document]
Comment (reindex-ot) [reply app/src/main/res/values-ja/strings.xml:391]: ❌️

File: app/src/main/res/values-ja/strings.xml (hunk 6 above)
[the remark this answers is not in the document]
Comment (reindex-ot) [reply app/src/main/res/values-ja/strings.xml:409]: ❌️

File: app/src/main/res/values-ja/strings.xml [hunk 8]
```diff
@@ -187,27 +227,35 @@
...
     <!-- Gallery Archives -->
-    <string name="no_archives">圧縮パッケージなし</string>
-    <string name="download_archive_started">圧縮パッケージのダウンロードを開始しました</string>
-    <string name="download_archive_failure">圧縮パッケージをダウンロードできませんでした</string>
-    <string name="download_archive_failure_no_hath">圧縮パッケージのダウンロードにはH@Hクライアントが必要です</string>
+    <string name="no_archives">アーカイブなし</string>
+    <string name="archive_original">オリジナル</string>
+    <string name="archive_resample">リサンプル</string>
+    <string name="archive_free">自由</string>
+    <string name="current_funds">Fund: %1$s - GP: %2$d クレジット</string>
```
[the remark this answers is not in the document]
Comment (reindex-ot) [reply app/src/main/res/values-ja/strings.xml]: update text.
` <string name="current_funds">Fund: %1$s | GP: %2$d Credit</string>`

File: app/src/main/res/values-ja/strings.xml [hunk 9]
```diff
@@ -262,205 +322,292 @@
...
     <string name="stat_download_done_title">ダウンロード完了</string>
-    <string name="stat_download_done_text_succeeded">%d件のダウンロード完了済み</string>
-    <string name="stat_download_done_text_failed">%d件のダウンロードにし失敗しました</string>
-    <string name="stat_download_done_text_mix">%1$d件のダウンロード完了済み、%2$d件に失敗</string>
-    <string name="stat_download_done_line_succeeded">ダウンロード完了：%s</string>
-    <string name="stat_download_done_line_failed">ダウンロード失敗：%s</string>
-
+    <string name="stat_download_done_text_succeeded">%d 件が成功</string>
+    <string name="stat_download_done_text_failed">%d 件が失敗</string>
+    <string name="stat_download_done_text_mix">%1$d 件のダウンロードが成功、%2$d 件が失敗</string>
```
[the remark this answers is not in the document]
Comment (reindex-ot) [reply app/src/main/res/values-ja/strings.xml]: update text.
````
    <string name="stat_download_done_text_succeeded">%d 件が成功しました</string>
    <string name="stat_download_done_text_failed">%d 件が失敗しました</string>
    <string name="stat_download_done_text_mix">%1$d 件のダウンロードが成功、%2$d 件が失敗しました</string>
````

File: app/src/main/res/values-ja/strings.xml (hunk 8 above)
[the remark this answers is not in the document]
Comment (xb2016) [reply app/src/main/res/values-ja/strings.xml]: Fund: %1$s GP | %2$d Credit

[the remark this answers is not in the document]
Comment (reindex-ot) [reply app/src/main/res/values-ja/strings.xml]: Changed
````
    <string name="current_funds">資金: %1$s | GP: %2$d Credits</string>
    <string name="insufficient_funds">資金 (GP) が不足しています</string>
````

[the remark this answers is not in the document]
Comment (reindex-ot) [reply app/src/main/res/values-ja/strings.xml]: Pipe removed.

[the remark this answers is not in the document]
Comment (reindex-ot) [reply app/src/main/res/values-ja/strings.xml]: Restore text.
```
    <string name="current_funds">Funds: %1$s GP, %2$d Credits</string>
    <string name="insufficient_funds">Funds (GP) が不足しています</string>
````

**Pull request merged**